test/integration/app/trigger/trigger.fn.spec.ts

```diff
@@ -96,7 +101,141 @@ const getWalletState = async (walletId: WalletId): Promise<WalletState> => {
 }
 
 describe("onchainBlockEventHandler", () => {
-  it("should process block for incoming transactions", async () => {
+  const waitForSettledTxs = async ({ wallets, addresses }): Promise<true> => {
+    return new Promise<true>(async (resolve) => {
+      // eslint-disable-next-line no-constant-condition
+      while (true) {
```

Comment:
Good point, we do already have an e2e that does exactly this at `test/e2e/servers/trigger/trigger.spec.ts`.

Do you think I can remove the `"should process pending invoices on invoice update event"` test as well since this is probably already covered in receive tests?

Reply:
you are right, already covered in receive, you can remove it if subscription is covered

Reply:
Done https://github.com/GaloyMoney/galoy/pull/2662/commits/880fdddf90f97a75e5a6ef5b4b73b2da21a92e71